### PR TITLE
feat: SingleProject - SimpleCalc

### PR DESCRIPTION
**Pull request by vatsashah45**

### CODE REVIEW

**Comment by eriklimakc** [comment]
It would be nice if we had a readme file similar to what we have for ChatGPT: https://github.com/unoplatform/Uno.Samples/tree/master/UI/ChatGPT

 - Intro
 - Screenshot
 - Getting Started (if anything needs to be setup like the API Key in TubePlayer - https://github.com/unoplatform/Uno.Samples/pull/640)
 - Code base (since SimpleCalc and Counter app has multiple versions, no need to specify files extensions like .xaml or .cs)
 - Docs to key features like (MVUX, C# Markup, etc) 

No need for a Architecture Diagram though (IMO).

And same for Counter App - https://github.com/unoplatform/Uno.Samples/pull/639

**Comment by eriklimakc** [comment]
@vatsashah45 I think we're missing the other variations of SimpleCalc. I only see CSharp + MVVM.

**Comment by eriklimakc** [comment]
We need to bring back Unit tests for SimpleCalc.

File: reference/SimpleCalc/CSharp-MVUX/SimpleCalculator/SimpleCalculator.csproj [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <PropertyGroup>
+    <TargetFrameworks>
+      net8.0-browserwasm;
+      net8.0-android;
+      net8.0-ios;
+      net8.0-maccatalyst;
+      net8.0-windows10.0.19041;
+      net8.0-desktop;
+      net8.0;
+    </TargetFrameworks>
```
Comment (eriklimakc) [inline reference/SimpleCalc/CSharp-MVUX/SimpleCalculator/SimpleCalculator.csproj:11]:
```suggestion
    <TargetFrameworks>
      net8.0-desktop;
      net8.0-android;
      net8.0-ios;
      net8.0-maccatalyst;
      net8.0-windows10.0.19041;
      net8.0-browserwasm;
    </TargetFrameworks>
```

File: reference/SimpleCalc/CSharp-MVVM/SimpleCalculator/SimpleCalculator.csproj [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  <PropertyGroup>
+    <TargetFrameworks>
+      net8.0-browserwasm;
+      net8.0-android;
+      net8.0-ios;
+      net8.0-maccatalyst;
+      net8.0-windows10.0.19041;
+      net8.0-desktop;
+      net8.0;
+    </TargetFrameworks>
```
Comment (eriklimakc) [inline reference/SimpleCalc/CSharp-MVVM/SimpleCalculator/SimpleCalculator.csproj:11]:
```suggestion
    <TargetFrameworks>
      net8.0-desktop;
      net8.0-android;
      net8.0-ios;
      net8.0-maccatalyst;
      net8.0-windows10.0.19041;
      net8.0-browserwasm;
    </TargetFrameworks>
```

File: reference/SimpleCalc/XAML-MVUX/SimpleCalculator/SimpleCalculator.csproj [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  <PropertyGroup>
+    <TargetFrameworks>
+      net8.0-browserwasm;
+      net8.0-android;
+      net8.0-ios;
+      net8.0-maccatalyst;
+      net8.0-windows10.0.19041;
+      net8.0-desktop;
+      net8.0;
+    </TargetFrameworks>
```
Comment (eriklimakc) [inline reference/SimpleCalc/XAML-MVUX/SimpleCalculator/SimpleCalculator.csproj:11]:
```suggestion
    <TargetFrameworks>
      net8.0-desktop;
      net8.0-android;
      net8.0-ios;
      net8.0-maccatalyst;
      net8.0-windows10.0.19041;
      net8.0-browserwasm;
    </TargetFrameworks>
```

File: reference/SimpleCalc/XAML-MVVM/SimpleCalculator/SimpleCalculator.csproj [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  <PropertyGroup>
+    <TargetFrameworks>
+      net8.0-browserwasm;
+      net8.0-android;
+      net8.0-ios;
+      net8.0-maccatalyst;
+      net8.0-windows10.0.19041;
+      net8.0-desktop;
+      net8.0;
+    </TargetFrameworks>
```
Comment (eriklimakc) [inline reference/SimpleCalc/XAML-MVVM/SimpleCalculator/SimpleCalculator.csproj:11]:
```suggestion
    <TargetFrameworks>
      net8.0-desktop;
      net8.0-android;
      net8.0-ios;
      net8.0-maccatalyst;
      net8.0-windows10.0.19041;
      net8.0-browserwasm;
    </TargetFrameworks>
```

File: reference/SimpleCalc/XAML-MVUX/SimpleCalculator/SimpleCalculator.csproj [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+    <OutputType>Exe</OutputType>
+    <SingleProject>true</SingleProject>
```
Comment (eriklimakc) [inline reference/SimpleCalc/XAML-MVUX/SimpleCalculator/SimpleCalculator.csproj:14]:
```suggestion
    <UnoSingleProject>true</UnoSingleProject>
```

File: reference/SimpleCalc/XAML-MVVM/SimpleCalculator/SimpleCalculator.csproj [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+    <OutputType>Exe</OutputType>
+    <SingleProject>true</SingleProject>
```
Comment (eriklimakc) [inline reference/SimpleCalc/XAML-MVVM/SimpleCalculator/SimpleCalculator.csproj:14]:
```suggestion
    <UnoSingleProject>true</UnoSingleProject>
```

**Review by eriklimakc — CHANGES_REQUESTED** [review]

**Comment by agneszitte** [comment]
Closing this PR in favor of https://github.com/unoplatform/Uno.Samples/pull/671

**Pull request closed without merging**